scipy-1.0/paper.tex

```diff
@@ -461,19 +461,16 @@ \section*{Discussion}
 consists of 610 unique contributors of source code, with $>19,000$ commits
 accepted into the code base (GitHub page data).
 
-% need some better usage stats sources?
-% SciPy stuff from njs blog: https://docs.google.com/spreadsheets/d/1lOLvSF0up4eZyv2ugZi-TM_GIs3gPCkGNIDKXS1Y3w4/edit#gid=406031940
-From the user side, there have been at least 15 million downloads
-of the SciPy manylinux wheels between the start of 2016 and May of 2018
-(njs blog). The recommended source for citing SciPy is simply a reference
-to the website / library itself\cite{SciPylib}, and has been cited $>3000$
-times. Some of the most prominent citing articles include the IPython
-publication\cite{PER-GRA:2007}, a heavily-cited Nature Biotechnology
-article\cite{pmid17287757}, a dark
-matter annihilitation study in the field of
-cosmology\cite{1475-7516-2012-08-007}, and many of the
-other major downstream software libraries in the Python scientific
-computing stack.
+From the user side, there were 13,096,468 PyPI downloads
+of SciPy during the year 2017\cite{pypinfo}, which is likely a vast
+underestimate of total downloads given that we don't have access to the
+popular Anaconda package manager download statistics. The recommended 
+source for citing SciPy is simply a reference to the website / library
+itself\cite{SciPylib}, and has been cited $>3000$ times. Some of the most
+prominent usages of or demonstrations of credibility for SciPy include the LIGO / Virgo
+scientific collaboration that lead to the observation of gravitational
+waves \cite{PhysRevLett.116.061102}, and the fact that SciPy is shipped directly
+with MacOS and in the Intel Distribution for Python\cite{intel-python}. 
```

Comment:
These are really strong!

For those (few) who are not already aware that SciPy is part of the Anaconda distribution of Python, the part earlier about not having access to the Anaconda download statistics might be confusing. 

For those who are aware, it might seem missing from this list at the bottom.

Reply:
Thoughts here: this is for a general journal, so mostly non-users. Intel is a name anyone recognizes, while Anaconda is not. Hence the choice for Intel. People who do use Python will already know that Anaconda is the most popular distribution

Reply:
Agreed... but these are precisely the reasons I thought changes might be appropriate. Just to clarify my reasoning:

"this is for a general journal, so mostly non-users"
This is why I thought it might be confusing above when it says "we don't have access to the popular Anaconda package manager download statistics". I understand that non-users are supposed to infer from this that Anaconda is a very common way for people to install SciPy, I'm just not sure that they will.

"People who do use Python will already know that Anaconda is the most popular distribution" 
This is why I thought that the fact that SciPy is included in Anaconda is such a good "demonstration of credibility", especially with Anaconda being mentioned earlier. Both Intel and Anaconda could be listed here,

Reply:
how about we just rephrase the download statistics part as "underestimate of total downloads given that downloading from PyPI is only one of several popular methods to install SciPy".

Reply:
Sure

Reply:
done in gh-59